api/ip-api.yaml

```diff
@@ -0,0 +1,280 @@
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Full set of package properties
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/informationPackage'
+              examples:
+                package:
+                  value: |-
+                    {
+                      “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b609”,
+                      “name”: “IP.101.XYZ”,
+                    }
+    delete:
+      tags:
+        - /ips
+      description: Delete information package binary by id.
+      summary: Delete IP by id.
+      operationId: deleteIpById
+      parameters:
+        - in: path
+          name: packageId
+          description: UUID of the package to delete
+          required: true
+          schema:
+            type: string
+      responses:
+        '201':
+           description: Package deleted
+           content:
+             application/json:
+               schema:
+                 type: object
+                 $ref: '#/components/schemas/packageId'
+               examples:
+                packageId:
+                  value: |-
+                    {
+                      “uid”: “845872bf-cbf8-48e6-b495-ec6b4a63b609”,
+                      “name”: “IP.101.XYZ”,
+                    }
+components:
+  schemas:
+    packageId:
+      type: object
+      properties:
+        uid:
+          type: string
+          nullable: false
+          format: uuid
+        name:
+          type: string
+          nullable: false
+    packageDetails:
+      type: object
+      properties:
+        id:
+          type: object
+          $ref: '#/components/schemas/packageId'
+        checksums:
+          type: array
+          items:
+            $ref: '#/components/schemas/digest'
+    digest:
+      type: object
+      properties:
+        algorithm:
+          type: string
+          nullable: false
+          default: SHA1
+          enum:
+            - MD5
+            - SHA1
+            - SHA256
+            - SHA512
+        value:
+          type: string
+          nullable: false
+    testResult:
+      type: object
+      properties:
+        ruleId:
+          type: string
+          nullable: false
+        location:
+          type: string
+          nullable: false
+        message:
+          type: string
+        severity:
+          type: string
+          enum:
+            - Info
+            - Warn
+            - Error
+    profileDetails:
+      type: object
+      properties:
+        id:
+          type: string
+          enum:
+            - CSIP
+            - SIP
+            - DIP
+        name:
+          type: string
+          nullable: false
+        version:
+          type: string
+          nullable: false
+    informationPackage:
+      type: object
+      properties:
+        details:
```

Comment:
Perhaps we should call it identification? Moreover, with this we introduce a consistency problem, as in some endpoints of /ips we simple return this information without a wrapper (i.e. details, identification, whatever we will call it)...

Reply:
Fix with envelope entity?